_tools/release_docs/api-reference-process.go

```diff
@@ -66,44 +42,59 @@ func addHeader(filename string) error {
 	if err != nil {
 		return err
 	}
-	//<head>
-	htmlNode2 := target.FirstChild.NextSibling
-
-	headNode2 := htmlNode2.FirstChild
-
-	//append source <head> after target <head>
-	detachNode(headNode)
-	htmlNode2.InsertBefore(headNode, headNode2)
-	bodyNode2 := headNode2.NextSibling.NextSibling
-	//insert nav header
-	detachNode(headerNode)
-	bodyNode2.InsertBefore(headerNode, bodyNode2.FirstChild)
-	//insert navContainerDivNode as target bodyNode's first child. Need to clean navContainerDivNode's Parent and Siblings first.
-	detachNode(navContainerDivNode)
-	bodyNode2.InsertBefore(navContainerDivNode, bodyNode2.FirstChild)
+	//node for <head>
+	headNode := target.FirstChild.NextSibling.FirstChild
 
-	if err := targetHTML.Truncate(0); err != nil {
+	err = targetHTML.Truncate(0)
+	if err != nil {
 		return err
 	}
-	if _, err := targetHTML.Seek(0, 0); err != nil {
+	_, err = targetHTML.Seek(0, 0)
+	if err != nil {
 		return err
 	}
-	err = html.Render(targetHTML, target)
-	return err
+	for node := headNode; node != nil; node = node.NextSibling {
+		err = html.Render(targetHTML, node)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
 }
 
-//TODO: These really should be fixed when generating the original html files
-func fixHeadAlign(filename string) error {
+func nitFix(filename string) error {
 	file, err := ioutil.ReadFile(filename)
 	if err != nil {
 		return err
 	}
-	newContents := strings.Replace(string(file), "h2 { font-size: 2.3125em; }", "h2 { font-size: 2.3125em; text-align: left;}", 1)
-	newContents = strings.Replace(newContents, "h4 { font-size: 1.4375em; } }", "h4 { font-size: 1.4375em; text-align: left;} }", 1)
-	newContents = strings.Replace(newContents, "<h2 id=\"_paths\">Paths</h2>", "<h2 id=\"_paths\">Operations</h2>", 1)
+	//TODO: This really should be fixed when generating the original html files
+	newContents := strings.Replace(string(file), "<h2 id=\"_paths\">Paths</h2>", "<h2 id=\"_paths\">Operations</h2>", 1)
+	//Adapt the color of headings to k8s style
+	newContents = strings.Replace(newContents, "color: #ba3925", "color: #428BCB", -1)
```

Comment:
The color is defined in the style of hmtl files in the master branch. We need some of the style to keep the table format. I think we should look for a way to overwrite these using _layouts/ and _scss/.